### PR TITLE
IDEM-172

### PR DESCRIPTION
**Pull request by bitcoinbrisbane**
Run unit test on push

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -6,11 +6,11 @@ on:
       - main
   push:
     branches:
-      - main
+      - IDEM-172
```
Comment (xcjlo985) [inline .github/workflows/ci.yml:9]: Presume you want to revert this before merging

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -6,11 +6,11 @@ on:
...
 
 jobs:
   build-and-lint:
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-20.04
```
Comment (xcjlo985) [inline .github/workflows/ci.yml:13]: I also presume if this runs fine we should update our standard to 20

File: package.json [hunk 3]
```diff
@@ -31,7 +32,9 @@
     "typescript": "^4.2.3"
   },
   "dependencies": {
+    "mocha": "^10.0.0",
```
Comment (xcjlo985) [inline package.json:35]: Dev dependency?

**Review by xcjlo985 — COMMENTED** [review]
I'll approve once the static checks pass. I checked this out and ran the unit tests and they are failing on my machine

File: .github/workflows/ci.yml (hunk 2 above)
Comment (bitcoinbrisbane) [reply .github/workflows/ci.yml:13]: Yep, agreed.

File: package.json (hunk 3 above)
Comment (bitcoinbrisbane) [reply package.json:35]: Fixed.

**Review by xiangzhe0812 — APPROVED** [review]

**Review by xcjlo985 — APPROVED** [review]

**Pull request merged**